doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/examples.md

```diff
@@ -5,7 +5,13 @@
 :::{note}
 To learn the basics of Ray on Kubernetes, we recommend taking a look
 at the {ref}`introductory guide<kuberay-quickstart>` first.
+
+If you are new to your cloud provider's Kubernetes service, we recommend
+taking a look at the documentation links collected {ref}`here<kuberay-k8s-setup>`
 :::
 
 This section presents example Ray workloads try out on your Kubernetes cluster.
```

Comment:
I know it's not this PR, but it should be `example Ray workloads _to_ try out on your`

Reply:
thanks! fixed.